### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by totaltrash** [bonfire-networks/iconify_ex#1]
Hi, first of all thank you so much for this library. I use Iconify on js projects, it's so nice to have it in Phoenix too.

When I installed using `{:iconify_ex, "~> 0.1.0"}`, I think I must have got an older version of iconfy_ex, as there was no assets directory. I had to install with `{:iconify_ex, github: "bonfire-networks/iconify_ex", branch: "main"}`. There aren't any tagged version on github (I'm not sure how releases in Hex hook up with tags on gh). Maybe it needs a new release or something?

**Comment by mayel** [comment]
I have published a new version: https://hex.pm/packages/iconify_ex
Please let me know if you encounter any issues or have feedback.

**Comment by totaltrash** [comment]
Yeah, something not quite right for me - I don't get any assets directory when installing that version either. I created a new app to test it on:

```
mix phx.new --no-assets iconify
```

Added `{:iconify_ex, "~> 0.2.0"}` to deps and then `mix deps.get`

Then:

```
ls -al deps/iconify_ex
total 28
drwxrwxr-x  3 dblack dblack 4096 Jul  4 10:39 .
drwxrwxr-x 42 dblack dblack 4096 Jul  4 10:39 ..
-rw-rw-r--  1 dblack dblack    0 Jul  4 10:39 .fetch
-rw-rw-r--  1 dblack dblack  274 Jul  4 10:39 .hex
-rw-rw-r--  1 dblack dblack 1277 Jul  4 10:39 hex_metadata.config
drwxrwxr-x  2 dblack dblack 4096 Jul  4 10:39 lib
-rw-r--r--  1 dblack dblack 1579 Jul  4 10:39 mix.exs
-rw-r--r--  1 dblack dblack 2721 Jul  4 10:39 README.md
```

To confirm I've git the updated `mix.exs`

```
cat deps/iconify_ex/mix.exs | grep @version
  @version "0.2.0"
```

Then changed the dep to `{:iconify_ex, github: "bonfire-networks/iconify_ex", branch: "main"}`, assets is there

```
ls -al deps/iconify_ex
total 44
drwxrwxr-x  5 dblack dblack 4096 Jul  4 10:46 .
drwxrwxr-x 42 dblack dblack 4096 Jul  4 10:46 ..
drwxrwxr-x  2 dblack dblack 4096 Jul  4 10:46 assets
-rw-rw-r--  1 dblack dblack    0 Jul  4 10:46 .fetch
drwxrwxr-x  8 dblack dblack 4096 Jul  4 10:46 .git
-rw-rw-r--  1 dblack dblack  716 Jul  4 10:46 .gitignore
drwxrwxr-x  2 dblack dblack 4096 Jul  4 10:46 lib
-rw-rw-r--  1 dblack dblack 1579 Jul  4 10:46 mix.exs
-rw-rw-r--  1 dblack dblack 8306 Jul  4 10:46 mix.lock
-rw-rw-r--  1 dblack dblack 2721 Jul  4 10:46 README.md
```

Strange huh?

**Comment by totaltrash** [comment]
I got it I think... `mix hex.build` doesn't include `assets` directory in the files option: https://hexdocs.pm/hex/Mix.Tasks.Hex.Build.html#module-package-configuration

**Comment by mayel** [comment]
@totaltrash aha! thanks for looking into it, I've pushed a new release and hope this works!

**Comment by totaltrash** [comment]
@mayel, yes, fixed in 0.2.1
